administrator/components/com_admin/sql/updates/postgresql/5.0.0-2023-03-17.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM "#__extensions" WHERE name = 'plg_task_demotasks';
```

Comment:
```suggestion
DELETE FROM "#__extensions" WHERE "type" = 'plugin' AND "element" = 'demotasks' AND "folder" = 'task' AND "client_id" = 0;
```
Explanation see my suggestion for the MySQL update SQL script.